polkadot/xcm/src/v3/junction.rs

```diff
@@ -126,7 +128,8 @@ impl From<NewNetworkId> for NetworkId {
 			BitcoinCore => Self::BitcoinCore,
 			BitcoinCash => Self::BitcoinCash,
 			PolkadotBulletin => Self::PolkadotBulletin,
-		}
+			Paseo => return Err(()),
```

Comment:
@franciscoaguirre i'm struggling to understand the logic here: Why Paseo would return an error?